code-challenges.test.js

```diff
@@ -23,11 +44,38 @@ const colors2 = ["chartreuse", "indigo", "periwinkle", "ochre", "aquamarine", "s
 
 // b) Create the function that makes the test pass.
 
+//Create a function named kolors that will shuffle an array and remove the first item
+// Use the method .shift to remove the first item of the array
+// Return the array with the method sort to sort all the values insde of the array
+/////// The method sort accepts a function that returns a value between -0.5 and 0.5
+
+
+const kolors = (array) => {
+  array.shift()
+      return array.sort((a, b) => Math.random() - 0.6)
+}
+
+console.log(kolors(colors1))
```

Comment:
When using the jest test suite you do not need console.log

If you were using it for troubleshooting then comment it out before sending up